Wrappers/Python/cil/plugins/tigre/Geometry.py

```diff
@@ -159,6 +157,10 @@ def __init__(self, ig, ag):
             pitch = np.arcsin(V[2])
             yaw = np.arctan2(-U[2],U[1])
 
+        #shift origin to match image geometry
+        self.offOrigin[1] += ig.center_y
+        self.offOrigin[2] += ig.center_x
+
```

Comment:
why have you moved this to here?

Reply:
It wasn't moved, it was just a bug and not included! This is the x-y shift which needs to happen for 2D and 3D cases.

Reply:
The 2D geometry wasn't taking in to account the centre offsets. So x and y offsets are moved to a common area from the 3D section.